gatsby/gatsby-plugin-sitemap/gatsby-plugin-sitemap.js

```diff
@@ -129,13 +129,20 @@ function gatsbyPluginSitemap({ allSitePage: { nodes: allPages } }) {
       lang: languageKey,
       path,
       links,
+      publicationDate: pageContext?.publicationDate,
     };
   });
 }
 
 function serialize(sitemapItem) {
+  const HIGH_PRIO =
+    sitemapItem.path.includes('/blog') ||
+    sitemapItem.path.includes('/career') ||
+    sitemapItem.path === '/' ||
+    sitemapItem.path === '/de';
```

Comment:
this is kind of self explaining but I feel somehow surprised by that because I get the reasoning only by some thinking. It's the update frequency mostly? Maybe a comment could provide that link why those are high prio ? Or use a name "FREQUENTLY_UPDATED" and use that variable for the priority? That would explain it very much to me.

Reply:
Yep, `FREQUENTLY_UPDATED` describes it better, thanks for that 👍  I was first using it for `prioity`, but then decided to only use it for the frequency, hence the "outdated" name.